src/object/object_printer.cpp

```diff
@@ -43,6 +43,30 @@
 
 #include <assert.h>
 
+//--------------------------------------------------------------------------------
+void object_printer::describe_comment_for_session_cmd (const char *comment)
+{
+  db_value comment_value;
+
+  assert (comment != NULL);
+
+  db_make_null (&comment_value);
+  db_make_string (&comment_value, comment);
+
+  m_buf ("COMMENT ");
+  if (comment != NULL && comment[0] != '\0')
+    {
+      db_value_printer printer (m_buf);
+      printer.describe_comment_value (&comment_value);
+    }
+  else
+    {
+      m_buf ("''");
+    }
+
+  pr_clear_value (&comment_value);
+}
+
```

Comment:
Most of the lines are duplicated with describe_comment(). How about just adding `class_description::type prt_type` as an argument in describe_comment () and calling describe_comment_value() or describe_value() according to the `prt_type`?

Reply:
I keep this change for simplify.